new-packaging/README.md

```diff
@@ -0,0 +1,34 @@
+## Pre-Requisites
+You need `xmlstarlet`, `dos2unix`
+
+You also need `cypher-shell` packaged
+
+You can either checkout `cypher-shell` and build it yourself:
```

Comment:
Needs link to cypher-shell. Don't assume people know what cypher-shell is ;)

I might even go this far

```
cd /tmp
git clone https://github.com/neo4j/cypher-shell.git
make out/cypher-shell.zip
cp out/cypher-shell.zip /path/to/neo4j
```